### ISSUE TITLE
Store user/group metadata changes

### ISSUE THREAD

**Issue by luckydonald** [fabianonline/telegram_backup#36]
Like name changes changes,
Profile picture changes (are they even stored somewhere?)

Not sure if telegram gives all this needed data chronologically (i think only chat renames), but storing that each run would be still helpful. (E.g. Deleted accounts, etc)

**Comment by luckydonald** [comment]
Or is that just not present in the web export?

**Comment by fabianonline** [comment]
Nope, that doesn't get saved. The tool only keeps the most recent data. But it's definitely a nice idea to collect this kind of data. I'll probably add this some time I've got some time to spare. ;-)

**Comment by luckydonald** [comment]
Some more thoughts:
Probably related to this would be edits of messages. Not sure if they are receivable on a later time.
Maybe note that the message was edited.
